client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java

```diff
@@ -1050,17 +1064,21 @@ public void testUpdateByQuery() throws Exception {
             long bulkRetries = bulkResponse.getBulkRetries(); // <9>
             long searchRetries = bulkResponse.getSearchRetries(); // <10>
             TimeValue throttledMillis = bulkResponse.getStatus().getThrottled(); // <11>
-            TimeValue throttledUntilMillis = bulkResponse.getStatus().getThrottledUntil(); // <12>
-            List<ScrollableHitSource.SearchFailure> searchFailures = bulkResponse.getSearchFailures(); // <13>
-            List<BulkItemResponse.Failure> bulkFailures = bulkResponse.getBulkFailures(); // <14>
+            TimeValue throttledUntilMillis =
+                    bulkResponse.getStatus().getThrottledUntil(); // <12>
+            List<ScrollableHitSource.SearchFailure> searchFailures =
+                    bulkResponse.getSearchFailures(); // <13>
+            List<BulkItemResponse.Failure> bulkFailures =
+                    bulkResponse.getBulkFailures(); // <14>
             // end::update-by-query-response
         }
         {
             UpdateByQueryRequest request = new UpdateByQueryRequest();
             request.indices("source1");
 
+            ActionListener<BulkByScrollResponse> listener;
```

Comment:
Kind of unfortunate that we have to do things like this to avoid long lines, but so be it.